### DISCUSSION TITLE
Problems with chat display

### DISCUSSION THREAD

**Discussion by yarosh69** [ORelio/Minecraft-Console-Client#2366] [Q&A]
The text is divided into [0m], which makes it difficult to read and at the same time does not allow using AutoRespond
How can this be fixed? win7 64
![ScreenShot_20221209121032](https://user-images.githubusercontent.com/120141684/206666834-6c43c35b-9490-4dd8-be7d-f80def22937a.png)

**Comment by BruceChenQAQ** [comment]
This is because the Win7 terminal does not support color escape codes. You can install a new terminal and run MCC with it.
For example, use Cmder (ConEmu).

**Reply by yarosh69** [reply]
I can't figure out how to run MCC in a new console. What needs to be do?

**Reply by ReinforceZwei** [reply]
Start your new terminal and drag MCC to it. Press enter to run.

**Reply by milutinke** [reply]
I'd suggest using Alacritty
https://alacritty.org/

**Reply by yarosh69** [reply]
![ScreenShot_20221210130910](https://user-images.githubusercontent.com/120141684/206849321-579c8962-0bbb-4945-87d0-773e45f7c285.png)
I found it, thanks
